### PR TITLE
Add user page.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Add pages for any user (with or without an account). This will look up their details on GitHub, display if they’ve joined the program and, if so, their contributions on Fridays for the last year.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Screenshot:
screen shot 2017-04-20 at 12 04 10

CC @nayafia @kytrinyx @bkeepers for thoughts on what else might be 🆒 to display here (or what we should consider removing).

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class UsersController < ApplicationController
+  def show
+    github_user = octokit.user params[:id]
+    @avatar = github_user.avatar_url
+    @name = github_user.name
+    @nickname = github_user.login
+    @name_or_nickname = @name || @nickname
+    @company = github_user.company
+    @location = github_user.location
+    @homepage = github_user.homepage
```
Comment (kytrinyx) [inline app/controllers/users_controller.rb]: How about the bio? Do we get that back from the API?

**Comment by MikeMcQuaid** [comment]
@kytrinyx 👍 will add bio in follow-up PR.

**Comment by nayafia** [comment]
Sorry I missed this! Looks good. I'll open an issue with more followups.

**Pull request merged**